mm2src/coins/nft/nft_structs.rs

```diff
@@ -118,18 +149,17 @@ pub struct NftList {
     pub(crate) nfts: Vec<Nft>,
 }
 
-#[allow(dead_code)]
 #[derive(Clone, Deserialize)]
 pub struct WithdrawErc1155 {
     pub(crate) chain: Chain,
-    from: String,
-    to: String,
-    token_address: String,
-    token_id: BigDecimal,
-    amount: BigDecimal,
+    pub(crate) from: String,
```

Comment:
Do we really need the `from` field in the request? we can use `my_address` fn straight away as it's used for `eth` withdraw https://github.com/KomodoPlatform/atomicDEX-API/blob/2344ebb4cf8e040c84a67670a220f14014792163/mm2src/coins/eth.rs#L874
This will remove the need for this check and error https://github.com/KomodoPlatform/atomicDEX-API/blob/2344ebb4cf8e040c84a67670a220f14014792163/mm2src/coins/eth.rs#L5031-L5036

Reply:
Yeah, I understand your question. Had the same thoughts. 
I was thinking how this will work when GUI side will be implemented. So I expected that `from` filed on GUI will be taken from cached NFT info and this (by this I mean `from` param and `eth_coin.my_address != from_addr`) was just an additional check that `my_address` is an owner of NFT. What I mean is, when user choose the NFT for transferring, GUI side has the access to specific NFT information (token address, owner, token id), so they can provide such info.

But yeah, since that its not so necessary check (`safeTransferFrom` function will fail, if user isn't NFT owner), I can remove it and use `coin.my_address()` instead.

Reply:
Done